app.json

```diff
@@ -2,6 +2,7 @@
   "name":"vets-website",
   "scripts":{},
   "env":{
+    "INSTALL_HOOKS": "no"
```

Comment:
@jkassemi should the default be 'yes'? I'm guessing this is overriden in Heroku envs (that way developers wouldn't have to think about it)